pipeline_products_session/jwst-data-products-part1.ipynb

```diff
@@ -0,0 +1,961 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can examine the [contents of an existing model](https://jwst-pipeline.readthedocs.io/en/latest/jwst/datamodels/models.html#looking-at-the-contents-of-a-model) with the ```.info()``` attribute:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ramp = datamodels.RampModel()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ramp.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### B.<font color='white'>-</font>Format<a class=\"anchor\" id=\"model-format\"></a>\n",
+    "\n",
+    "Let's go back and examine our uncalibrated file, but this time we will use a JWST data model. Below, we access the typical FITS extensions using the model attributes. All data related to the product are contained in one or more data arrays, and the headers are stored in the model metadata.\n",
+    "\n",
+    "In the model info, you see familiar names: ```data``` (```SCI``` extension for FITS), ```zeroframe``` (```ZEROFRAME```), ```group``` (```GROUP```), etc.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with datamodels.open(uncal_obs) as model:\n",
+    "    model.info()\n",
+    "    \n",
+    "## or use a specific model:\n",
+    "\n",
+    "model = datamodels.RampModel(uncal_obs)\n",
+    "model.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The exposure data is accessed through the ```data``` member of the model, instead of, for instance, the ```SCI``` extension of a FITS file. So instead of:\n",
+    "\n",
+    "```python\n",
+    "hdulist['SCI'].data\n",
+    "```\n",
+    "\n",
+    "you would use:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "science_data = model.data\n",
+    "science_data.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This data can be used the same way as before:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "integration = 2\n",
+    "group = -1\n",
+    "\n",
+    "fig = plt.figure(figsize=(8, 8))\n",
+    "ax = plt.subplot()\n",
+    "plt.imshow(science_data[integration, group, :, :], origin='lower', cmap='gray', vmin=4000, vmax=12000)\n",
+    "plt.xlabel('Pixel column')\n",
+    "plt.ylabel('Pixel row')\n",
+    "fig.tight_layout()\n",
+    "plt.subplots_adjust(left=0.15)\n",
+    "plt.colorbar(label='DN')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### C.<font color='white'>-</font>Metadata<a class=\"anchor\" id=\"model-meta\"></a>"
```

Comment:
It might be useful to have a cell that shows the result from <model>.meta.instance in order to see all the metadata entries at once.

Reply:
Thank god, I could not figure out what that attribute was. Added an example